src/test/java/org/mvel2/util/StringAppenderTest.java

```diff
@@ -0,0 +1,109 @@
+package org.mvel2.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mvel2.util.StringAppender;
+
+public class StringAppenderTest {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
```

Comment:
Doesn't look like you're using this rule or it's associated imports

Reply:
Fixed.